18 Normalizr/src/reducers/domain/country/byId.ts

```diff
@@ -0,0 +1,18 @@
+import { CountryView } from '../../../model/view/countryView';
+
+export const byId = (state: {[id: number]: CountryView} = {}, action) => {
+  if (action.payload && action.payload.entities) {
+    return {
+      ...state,
+      ...action.payload.entities.countries
+    }
+  }
+
+  return state;
+}
+
+export const getCountry = (state, id): CountryView => {
+  return {
+    ...state.countryDomain.byId[id]
```

Comment:
The name `countryDomain` here seems like this method knows more than it should. It's just a minor hint, but may indicate that we're missing something.